server/router/driver.ts

```diff
@@ -12,6 +12,50 @@ import { UserType } from '../models/subscription';
 const router = express.Router();
 const tableName = 'Drivers';
 
+// Get all drivers
+router.get('/', validateUser('Admin'), (req, res) => {
+  db.getAll(res, Driver, tableName);
+});
+
+// Get all available drivers at a specific date and time
+router.get('/available', validateUser('User'), (req, res) => {
+  const { date, startTime: reqStartTime, endTime: reqEndTime } = req.query;
```

Comment:
since you repeatedly use `reqStartTime as string` and `reqEndTime as string` multiple times, you should create separate string variables for `reqStartTime` and `reqEndTime`

Reply:
Done